caddy/restartinproc.go

```diff
@@ -1,32 +1,40 @@
 package caddy
 
-import "log"
+import (
+	"log"
+	"os"
+)
+
+// File descriptors for in-process restart
+var restartFds map[string]*os.File = make(map[string]*os.File)
```

Comment:
No race condition.

Follow-up though, could we revise this line to remove the lint notice:

> caddy\restartinproc.go:9:16: should omit type map[string]*os.File from declaration of var restartFds; it will be inferred from the right-hand side

So, change to:

```go
var restartFds = make(map[string]*os.File)
```